runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java

```diff
@@ -28,11 +28,19 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 import net.runelite.client.config.Units;
 
 @ConfigGroup("agility")
 public interface AgilityConfig extends Config
 {
+	@ConfigSection(
+		name = "Hallowed Sepulchre",
```

Comment:
You already have the section name defined in a variable, use that here.
```suggestion
		name = seepulchreSection,
```

Reply:
That'd be an illegal forward reference